sw/device/tests/entropy_src_ast_rng_req_test.c

```diff
@@ -19,7 +19,7 @@ enum {
    * The size of the buffer used in firmware to process the entropy bits in
    * firmware override mode.
    */
-  kEntropyFifoBufferSize = 32,
+  kEntropyFifoBufferSize = 16,
```

Comment:
This constant is used to set `buffer_threshold` in `dif_entropy_src_fw_override_config_t`. In other tests, it's named `kEntropySrcFifoThreshold`, which I think is more appropriate. This shouldn't gate this PR, but it might be worth improving.

This also occurs in `sw/device/tests/entropy_src_fw_ovr_test.c` and `sw/device/tests/sim_dv/ast_clk_rst_inputs.c`.

Reply:
@h-filali: May I ask you to create a small PR to change this?

Reply:
I think the constant refers to the buffer used in SW. The contents of the FIFO get dumped into a buffer which is of the size kEntropyFifoBufferSize. The FIFO threshold is then set to the same size as the buffer. I also think the naming could be improved since there are places where the FIFO is called a HW buffer (not necessarily wrong but also not optimal). But I think we'd have to change a lot for little benefit.

Reply:
Ah okay, I didn't read the code so thoroughly. Thanks for the explanation; I'm fine with the status quo then